platform/core.multitabs/src/org/netbeans/core/multitabs/impl/TabDataRenderer.java

```diff
@@ -58,6 +58,7 @@
     private static final Color underlineColor = UIManager.getColor("nb.multitabs.underlineColor"); // NOI18N
     private static final Color inactiveUnderlineColor = UIManager.getColor("nb.multitabs.inactiveUnderlineColor"); // NOI18N
     private static final Color activeBackground = UIManager.getColor("nb.multitabs.activeBackground"); // NOI18N
+    private static final Color hoverBackground = UIManager.getColor("nb.multitabs.hoverBackground"); // NOI18N
```

Comment:
Other LAFs, e.g. Windows, have a hover color as well. Why does FlatLAF need a new property here?

If the new property is needed, should the other LAFs be updated to use it as well?

Reply:
> Other LAFs, e.g. Windows, have a hover color as well.

There are two implementation for editor tabs in NB:
- "Tab Control" (package org.netbeans.swing.tabcontrol)
- "Multi-tabs" (package org.netbeans.core.multitabs)

Tab-control is used by default and has hover colors in all Lafs (I think).
Multi-tabs is used e.g. if tab placement is left/right/bottom (see #1865 for more options to enable multi-tabs). Multi-tabs did not support hover in the past. This PR adds hover to multi-tabs.

> If the new property is needed, should the other LAFs be updated to use it as well?

Yes, this a good idea. Can be done in another PR.

Reply:
Ah, I wasn't aware that different tab positions used a different implementation. Indeed on Windows these do not seem to support hover.

Reply:
Wait, but the screenshots show the editor tabs in the "top" position? Isn't that a tabcontrol type component, rather than a multitabs?

Reply:
Multi-tabs are also used for top position if you enable one of the 4 checkboxes at the bottom of the screenshot below:

![image](https://user-images.githubusercontent.com/5604048/72222672-bd2d7d00-3567-11ea-8d83-2f3e777222e3.png)

The screenshot highlights all options that enable multi-tabs.

I think I had enabled "Sort opend docs by project" when capturing the screenshots. Then only the popup menu is changed, but the tabs do not show additional information and look identical to Tab-control in FlatLaf. In other Lafs the Multi-tabs always look slightly different to Tab-control because Tab-control has UI delegates for different LaFs.

Reply:
OK, that explains it. I was curious, since I was thinking of doing some tweaks to the Windows LAF tabcontrol myself at some point.